public/app/plugins/datasource/loki/components/LokiQueryField.tsx

```diff
@@ -183,6 +183,7 @@ export class LokiQueryField extends React.PureComponent<LokiQueryFieldProps, Lok
                       onChange={this.onChangeQuery}
                       onRunQuery={onRunQuery}
                       initialValue={query.expr ?? ''}
+                      placeholder="Enter a LogQL query."
```

Comment:
```suggestion
                      placeholder="Enter a Loki query (run with Shift+Enter)"
```

This used to be our original placeholder for this datasource.